### PR TITLE
DOC-2498: TinyMCE 7.5 Release notes and community changelog.

### PR DESCRIPTION
**Pull request by kemister85**
Ticket: DOC-2498

Site: [Staging branch](http://docs-feature-75-doc-2498.staging.tiny.cloud/docs/tinymce/latest/)

Changes:
* TinyMCE 7.5 release notes, and community changelog.

Pre-checks:
- [x] Branch prefixed with `feature/<version>/`, `hotfix/<version>/`, `staging/<version>/`, or `release/<version>/`.
- [x] `modules/ROOT/nav.adoc` has been updated `(if applicable)`.
- [x] Included a `release note` entry for any `New product features`.
- [x] If this is a minor release, updated `productminorversion` in `antora.yml` and added new supported versions entry in `modules/ROOT/partials/misc/supported-versions.adoc`.

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/pages/7.5-release-notes.adoc [hunk 1]
```diff
@@ -13,181 +13,254 @@ include::partial$misc/admon-releasenotes-for-stable.adoc[]
...
+{productname} {release-version} addresses this issue. Now, the `aria-label` attribute has been moved to the input field to ensure it is correctly associated. Additionally, the descriptive text was updated from "Red/Green/Blue component" to "Red/Green/Blue channel" to provide clearer information for assistive technologies. These changes ensure that screen readers consistently and accurately announce the relevant RGB channels, significantly improving accessibility.
 
 [[additions]]
 == Additions
 
 {productname} {release-version} also includes the following addition<s>:
 
-=== <TINY-vwxyz 1 changelog entry>
-// #TINY-vwxyz1
+=== Added new default `li` element styles to the content CSS to prevent child list elements from inheriting certain parent list item styles.
```
Comment (spocke) [inline modules/ROOT/pages/7.5-release-notes.adoc:193]: We decided to drop this see TINY-11217 ticket for details.

File: modules/ROOT/pages/7.5-release-notes.adoc [hunk 2]
```diff
@@ -13,181 +13,267 @@ include::partial$misc/admon-releasenotes-for-stable.adoc[]
...
-[[changes]]
-== Changes
+For more information on customizing the color picker, see xref:user-formatting-options.adoc#color_map_raw[`+color_map_raw+` option].
 
-{productname} {release-version} also includes the following change<s>:
+NOTE: {companyname} Technologies would like to thank link:https://github.com/andrewnicols[Andrew Nicols] for contributing to this feature.
 
-=== <TINY-vwxyz 1 changelog entry>
-// #TINY-vwxyz1
+=== Added new default `li` element styles to the content CSS to prevent child list elements from inheriting certain parent list item styles.
```
Comment (kemister85) [inline modules/ROOT/pages/7.5-release-notes.adoc:206]: Ticket was reverted, removing from release notes
```suggestion
```

File: modules/ROOT/pages/changelog.adoc [hunk 3]
```diff
@@ -4,9 +4,32 @@
...
-// Unreleased
-== 7.5 - 2024-11-06
-// TODO
+== 7.5.0 - 2024-11-06
+
+=== Added
+
+* Added support for using raw CSS in the list of possible colors, using the `color_map_raw` property.
+// #GH-9788
+* Added new default `li` element styles to the content CSS to prevent child list elements from inheriting certain parent list item styles.
```
[the remark this answers is not in the document]
Comment (kemister85) [reply modules/ROOT/pages/changelog.adoc:13]:
```suggestion
```

File: modules/ROOT/pages/7.5-release-notes.adoc [hunk 4]
```diff
@@ -13,181 +13,266 @@ include::partial$misc/admon-releasenotes-for-stable.adoc[]
...
+NOTE: {companyname} Technologies would like to thank link:https://github.com/andrewnicols[Andrew Nicols] for contributing to this feature.
 
-=== <TINY-vwxyz 1 changelog entry>
-// #TINY-vwxyz1
+// #TINY-11217
 
-// CCFR here.
+In previous versions of {productname}, child `+<li>+` elements could inherit inline CSS styles from their parent list items, causing unexpected styling behavior in nested lists. This issue was due to native CSS behavior, where child list items would inherit certain parent styles upon post-content insertion.
 
+{productname} {release-version} addresses this issue. Now, new default CSS styles were added for `+<li>+` elements to prevent inheritance of specific parent styles. This ensures that list items in nested structures render independently, maintaining consistent visual formatting.
```
Comment (kemister85) [inline modules/ROOT/pages/7.5-release-notes.adoc:210]:
```suggestion
```

File: modules/ROOT/pages/7.5-release-notes.adoc [hunk 5]
```diff
@@ -13,181 +13,263 @@ include::partial$misc/admon-releasenotes-for-stable.adoc[]
...
 
-{productname} {release-version} also includes the following change<s>:
+For more information on customizing the color picker, see xref:user-formatting-options.adoc#color_map_raw[`+color_map_raw+` option].
 
-=== <TINY-vwxyz 1 changelog entry>
-// #TINY-vwxyz1
+NOTE: {companyname} Technologies would like to thank link:https://github.com/andrewnicols[Andrew Nicols] for contributing to this feature.
 
-// CCFR here.
+// #TINY-11217
```
Comment (kemister85) [inline modules/ROOT/pages/7.5-release-notes.adoc:206]:
```suggestion
```

**Review by kimwoodfield — APPROVED** [review]

**Review by tiny-ben-tran — APPROVED** [review]

**Review by ltrouton — APPROVED** [review]

**Review by ShiridiGandham — APPROVED** [review]
LGTM

**Review by MitchC1999 — APPROVED** [review]

**Pull request merged**